.github/workflows/main.yml

```diff
@@ -112,7 +112,7 @@ jobs:
         run: |
           set -xeu
           python -VV
-          pip install setuptools wheel
+          pip install setuptools wheel "setuptools-scm[toml]"
```

Comment:
Is it needed ? I thought the point of putting that in pyproject.toml was that it was THE right place to put it.

Note: this applies to the whole line :)

Reply:
In my tests I had to add it, but I don't know exactly why the build-system was not used when building the sdist/bdist_wheel